go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/yuzutech/kroki-cli
```

Comment:
for a binary, it could just be `module kroki`. Then, instead of importing `github.com/yuzutech/kroki-cli/cmd` for example in `main.go`, you will be able to import `kroki/cmd`. The output of `go build` will also be `kroki` and not `kroki-cli`.

Reply:
Done